### ISSUE TITLE
add eladkarako's HOSTS/AdBlock 

### ISSUE THREAD

**Issue by DJCrashdummy** [collinbarrett/FilterLists#119]
the VIEW-/ADD-urls (404) and i guess also the name are outdated.

**Comment by collinbarrett** [comment]
Hey @eladkarako , could you provide the name, description, and URLs that you would prefer that we use to index your project on FilterLists? Thanks.

**Comment by eladkarako** [comment]
Hi @collinbarrett - sorry, names will be changed again soon,
so I would not recommend to add direct URLs to any lists just yet,
(and probably to remove the current ones).

Ps. I recommend that anyone need to use it,
may forked it and use the direct link from their own repository,
(anything there is in public domain).

**Comment by collinbarrett** [comment]
removed for now. @eladkarako , feel free to reopen or open a new issue if you ever want your lists on FilterLists again. Thanks!

**Comment by eladkarako** [comment]
@collinbarrett 
here are the updated links, of the entire <a href="http://hosts.eladkarako.com/">GitHub hosted</a> repository.

<pre>
HOSTS (just domains, no prefix)
https://raw.githubusercontent.com/eladkarako/hosts/master/_raw__hosts.txt

HOSTS (HOSTS format)
https://raw.githubusercontent.com/eladkarako/hosts/master/build/hosts.txt
https://raw.githubusercontent.com/eladkarako/hosts/master/build/hosts0.txt
https://raw.githubusercontent.com/eladkarako/hosts/master/build/hosts_with_localhost.txt
https://raw.githubusercontent.com/eladkarako/hosts/master/build/hosts0_with_localhost.txt

HOSTS (same but in a format compatible with AdBlock-browser extensions)
https://raw.githubusercontent.com/eladkarako/hosts/master/build/hosts_adblock.txt

AdBlock lists
https://raw.githubusercontent.com/eladkarako/hosts/master/build/hosts_adblock_anti_annoyances_block.txt
https://raw.githubusercontent.com/eladkarako/hosts/master/build/hosts_adblock_anti_annoyances_block_inline_script.txt
https://raw.githubusercontent.com/eladkarako/hosts/master/build/hosts_adblock_anti_annoyances_hide.txt
https://raw.githubusercontent.com/eladkarako/hosts/master/build/hosts_adblock_anti_annoyances_style_inject.txt
</pre>

<hr/>
TL;DR

<code>hosts_adblock_anti_annoyances_hide.txt</code> is a collection of hiding-rules,
<code>hosts_adblock_anti_annoyances_block.txt</code> is purely connection-blocking,

Those AdBlock-lists will <strong>work best with <a href="https://github.com/gorhill/uBlock/releases">uBlock-origin</a> <sup><a href="https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm">#1</a>, <a href="https://addons.mozilla.org/firefox/addon/ublock-origin/">#2</a></sup></strong> (and not other AdBlock extensions).

<code>hosts_adblock_anti_annoyances_block_inline_script.txt</code> designed to stop malicious-scripts before they are being executed at all,
<code>hosts_adblock_anti_annoyances_style_inject.txt</code> allow small fixes that can not be done by hiding-rules.

**Comment by DJCrashdummy** [comment]
sorry for hijacking this treat, but i don't know how else to contact @eladkarako:
your list "_Hide Annoying Elements​​​​​​​_" (hosts_adblock_anti_annoyances_hide.txt) blocks the duckduckgo.com-searchbar! :frowning_face:
